ember-cli-build.js

```diff
@@ -20,12 +22,5 @@ module.exports = function(defaults) {
   // please specify an object with the list of modules as keys
   // along with the exports of each module as its value.
 
-  //return app.toTree();
-  const Webpack = require('@embroider/webpack').Webpack;
-  return require('@embroider/compat').compatBuild(app, Webpack, {
-    staticAddonTrees: true,
-    staticAddonTestSupportTrees: true,
-    staticHelpers: true,
-    staticComponents: true
-  });
```

Comment:
Was it intended to remove the Embroider setup here?

Reply:
Yes. I remember it was causing problems, it is not in a default Ember app, and I didn't have another project to look at for the correct set-up, so I decided the easiest solution was to remove all of it.